ginga/examples/configs/channel_Image.cfg

```diff
@@ -0,0 +1,118 @@
+#
+# Image channel preferences file
+#
+# Place this in file under ~/.ginga with the name "channel_Image.cfg" for
+# channel name "Image". This file WILL BE OVERWRITTEN by the Preferences
+# plugin (see below). NOT ALL PARAMETERS IN GUI CAN BE ADJUSTED HERE.
+#
+# Alternately, the settings can be adjusted via Preferences local plugin
+# within Ginga. NOT ALL PARAMETERS CAN BE ADJUSTED VIA GUI but users can
+# normally ignore the hidden ones.
+#
+
+# ---------------
+# Color Distribution
+
+color_algorithm = 'linear'
+
+# ---------------
+# Color Mapping
+
+color_map = 'gray'
+color_hashsize = 65535
+intensity_map = 'ramp'
+
+# ---------------
+# Auto Cuts
+
+autocut_method = 'zscale'
+autocut_params = {}
```

Comment:
Hmm, I just checked and actually this is supposed to be a list of tuples.  Example might be:
```python
[('usecrop', True), ('numbins', 2048), ('pct', 0.993)]
```
if `autocut_method` is set to 'histogram'.

I think there is a bug in restoring the params from the saved settings--looking into that.